gst-play/gst-play.c

```diff
@@ -347,6 +566,15 @@ keyboard_cb (const gchar * key_input, gpointer user_data)
   GstPlay *play = (GstPlay *) user_data;
 
   switch (g_ascii_tolower (key_input[0])) {
+    case 'i':
+      {
+        GstPlayerMediaInfo *media_info = gst_player_get_media_info (play->player);
+        print_media_info (media_info);
```

Comment:
Only calls this if media_info!=NULL, then you only need that check in one place instead of two